tmva/pymva/test/generateKerasModelSequential.py

```diff
@@ -0,0 +1,22 @@
+import os
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+
+import keras
```

Comment:
Same comment as above

Reply:
Modified code to import from tensorflow.keras